core/src/main/java/io/questdb/cutlass/Services.java

```diff
@@ -153,11 +153,11 @@ public LineTcpReceiver createLineTcpReceiver(
         // - DEDICATED (1 worker) when ^ ^ is not set
         // - SHARED otherwise
 
-        final WorkerPool ioPool = workerPoolManager.getInstance(
+        final WorkerPool ioPool = workerPoolManager.getInstanceIO(
                 config.getIOWorkerPoolConfiguration(),
                 Requester.LINE_TCP_IO
         );
-        final WorkerPool writerPool = workerPoolManager.getInstance(
+        final WorkerPool writerPool = workerPoolManager.getInstanceWrite(
```

Comment:
naming, it does not immediately jump at me what the difference between "IO" pool and "Write" pool. It reads that one is a subset of the other